.mailmap

```diff
@@ -1569,3 +1569,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+zzc <1378113190@qq.com>
```

Comment:
After adding your name and email in `.mailmap` file you need to run the file by the following command `python bin/mailmap_check.py`.
 Read [instructions](https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file).

Reply:
get it